### DISCUSSION TITLE
Image Background and Transparency

### DISCUSSION THREAD

**Discussion by Wishdream** [Orama-Interactive/Pixelorama#576] [Ideas]
There's this function from Krita that could be useful for Pixelorama in which you can adjust a file's image background and transparency on top of the transparent checkerboard pattern.

*It's not a solid color layer*, but an adjustable color layer in which when fully transparent the checkerboard pattern shows otherwise, it shows the color.

![bg](https://user-images.githubusercontent.com/4660711/81119523-8556a500-8f5d-11ea-9c26-05b20c4c56db.gif)

**Comment by Erevoid** [comment]
![Capture](https://user-images.githubusercontent.com/58960464/81207933-04d48a80-8fd7-11ea-8939-4862b21d8adc.PNG)

I am not sure if I understood correctly, but there already is an option to change the checkerboard pattern colors. If the user wants a solid color they can make both colors the same and it would have the same effect. Is there something more to how krita does it?

**Comment by Wishdream** [comment]
Yes. The difference is that it is a per-image basis. And the options are separate.
For example, the `Image Background and Transparency` goes into the image options for quick access:

![image](https://user-images.githubusercontent.com/4660711/81234909-53f8da80-902c-11ea-9688-c82e3eb29baa.png)

While the checkerboard pattern is inside the settings and affects only the alpha texture.

![image](https://user-images.githubusercontent.com/4660711/81234974-7985e400-902c-11ea-946b-96d98e33b3d2.png)
